src/global/translations/en.json

```diff
@@ -6,6 +6,16 @@
     "wallet": "WALLET",
     "profile": "PROFILE"
   },
+  "activity_log": {
```

Comment:
Please use camel case to name fields.

Reply:
`CamelCase` or `lowerCamelCase`? camel case look's strange to me.

Reply:
Oh, sorry, I meant lowerCamelCase of course. Just as any other fields in that file.